src/locale/locales/ca/messages.po

```diff
@@ -3322,23 +3322,23 @@ msgstr "Obre la pantalla amb tots els canals desats"
 
 #: src/view/screens/Settings/index.tsx:696
 msgid "Opens the app password settings"
-msgstr ""
+msgstr "Obre la configuració de les contrasenyes d'apliació"
```

Comment:
Comparing to the superseded string below, I think there might be a typo in the last word with a missing `c` in `aplicació`